sunspot/lib/sunspot/query/sort.rb

```diff
@@ -134,6 +134,23 @@ class FunctionComp
         def initialize(setup,args)
           @function=args.shift
           @fields = []
+          @extras = []
+
+          case @function.to_s
+          when "strdist"
+            # Reference: http://wiki.apache.org/solr/FunctionQuery#strdist
+            valid_measures = [:jw, :edit, :ngram]
+            extra = args.pop
+            if valid_measures.include? extra.to_sym
+              @extras.push(extra)
+            else
+              raise(
+                UnrecognizedRestrictionError,
+                "Use only valid measure options with strdist (#{valid_measures.to_s})"
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Redundant use of `Object#to_s` in interpolation.](https://www.codacy.com/app/joni-amir/sunspot/file/2482635387/issues/source?bid=3265104&fileBranchId=3265139#l149)